internal/services/contracts_events_consumer.go

```diff
@@ -143,7 +143,7 @@ func (c *ContractsEventsConsumer) routeTransferEvent(e *ContractEventData) error
 		c.log.Debug().Str("event", e.EventName).Interface("fullEventData", e).Msg("Handler not provided for contract")
 	}
 
-	return errors.New("Handler not provided for contract")
+	return nil
```

Comment:
I sort of wanted to avoid "ignoring mint event" and "device has not been claimed yet". Those happen a lot.